.github/workflows/docker.yaml

```diff
@@ -0,0 +1,33 @@
+name: Build CLI Docker image
```

Comment:
what about https://goreleaser.com/customization/docker/

Reply:
Sounds good.
I totally missed that gorelaser files are present: 
https://github.com/mongodb/mongodb-atlas-cli/blob/50e37b59ba912d661654fd4cf8f152356da942c3/build/package/.atlascli.goreleaser.yml#L14